tools/RAiDER/getStationDelays.py

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/env python3
+#
+import copy
+import datetime as dt
+import numpy as np
+import os
+import pandas as pd
+
+def getDate(stationFile):
```

Comment:
Same comment here about ordering the functions from most to least important/first to last called.